portable/IAR/AtmelSAM7S64/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
+                                                unsigned int flag ) /* \arg  pio input filter to be disabled */
+    {
         pPio->PIO_IFDR = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_GetInputFilterStatus
-//* \brief Return PIO Input Filter Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_PIO_GetInputFilterStatus( // \return PIO Input Filter Status
-        AT91PS_PIO pPio) // \arg  pointer to a PIO controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetInputFilterStatus */
+/** \brief Return PIO Input Filter Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetInputFilterStatus( /* \return PIO Input Filter Status */
+        AT91PS_PIO pPio )                                 /* \arg  pointer to a PIO controller */
+    {
         return pPio->PIO_IFSR;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_IsInputFilterSet
-//* \brief Test if PIO Input filter is Set
-//*----------------------------------------------------------------------------
-__inline int AT91F_PIO_IsInputFilterSet(
-        AT91PS_PIO pPio,   // \arg  pointer to a PIO controller
-        unsigned int flag) // \arg  flag to be tested
-{
-        return (AT91F_PIO_GetInputFilterStatus(pPio) & flag);
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_GetOutputDataStatus
-//* \brief Return PIO Output Data Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_PIO_GetOutputDataStatus( // \return PIO Output Data Status
-    AT91PS_PIO pPio) // \arg  pointer to a PIO controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_IsInputFilterSet */
+/** \brief Test if PIO Input filter is Set */
+/**---------------------------------------------------------------------------- */
+    __inline int AT91F_PIO_IsInputFilterSet( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                             unsigned int flag ) /* \arg  flag to be tested */
+    {
+        return( AT91F_PIO_GetInputFilterStatus( pPio ) & flag );
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetOutputDataStatus */
+/** \brief Return PIO Output Data Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetOutputDataStatus( /* \return PIO Output Data Status */
+        AT91PS_PIO pPio )                                /* \arg  pointer to a PIO controller */
```

Comment:
Should this be one line?

Reply:
Answered [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313523527)